Tripler UI: show age, hide street number

cooperka opened the pull request:
Also makes the width more responsive and cleans up some code.

# /triplers

### Before

<img width="800" alt="Screen Shot 2020-11-14 at 17 12 37" src="https://user-images.githubusercontent.com/2047062/99160484-c235ee00-269c-11eb-8fb6-4f3d2ac6fb3e.png">

### After

<img width="800" alt="Screen Shot 2020-11-14 at 17 13 03" src="https://user-images.githubusercontent.com/2047062/99160485-c3ffb180-269c-11eb-8b09-c69983e4dbb9.png">

# /triplers/add

### Before

<img width="800" alt="Screen Shot 2020-11-15 at 10 20 31" src="https://user-images.githubusercontent.com/2047062/99193200-42effb00-272c-11eb-8c8f-63bc86599ab1.png">

### After

<img width="800" alt="Screen Shot 2020-11-15 at 10 28 57" src="https://user-images.githubusercontent.com/2047062/99193379-67000c00-272d-11eb-9cd9-d4753a10c3f5.png">

cooperka committed:
Clean up some TriplersPage filtering logic

cooperka committed:
Refactor AllTriplers component to modularize Triplers

cooperka committed:
Normalize formatting of Triplers.mocks.js

cooperka committed:
Include age_decade in normalizeTripler()

cooperka committed:
Be more responsive with tripler names and addresses

cooperka committed:
Add test for tripler normalization

cooperka committed:
Normalize addresses using addresser

shauminn-colab commented:
@cooperka -- hey there, these look great! This might just be a screenshot snafu, but it looks like the screenshot for /triplers/add is actually a mobile screenshot for /triplers. Just checking that /triplers/add also got this update? 

Also, I'm sorry I didn't catch this in advance, but maybe we should make the age range `(21-30 years of age)` or `(Age: 21-30)` to be clearer? What do you think?

cooperka commented:
Oops, I fixed the /triplers/add screenshots!

And added `Age:` to both routes; the screenshot for /triplers doesn't reflect that change but they use the exact same code.

cooperka committed:
Show 'Age:' for clarity

shauminn-colab commented:
you're the best, thanks so much again!
